app/models/concerns/gias_helpers.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module GiasHelpers
+  extend ActiveSupport::Concern
+  include GiasTypes
+
+  included do
+    scope :currently_open, -> { where(school_status_code: GiasTypes::ELIGIBLE_STATUS_CODES) }
+    scope :eligible_establishment_type, -> { where(school_type_code: GiasTypes::ELIGIBLE_TYPE_CODES) }
+    scope :in_england, -> { where("administrative_district_code ILIKE 'E%'") }
+    scope :eligible, -> { currently_open.eligible_establishment_type.in_england }
+
+    enum school_status_name: {
+      open: "Open",
+      closed: "Closed",
+      proposed_to_close: "Open, but proposed to close",
+      proposed_to_open: "Proposed to open",
+    }, _suffix: "status"
```

Comment:
I find it surprising that we do open on code and closed on name

Reply:
I did think we should look at doing away with the code attribute and simplify this to just an enum with open or closed as that's all we're really interested in (i.e. `open` and `proposed_to_close` become simply `open` and the other two become `closed`) and we could handle that during import to the data stage?

Reply:
I used code because I didn't trust the format of the name field. If you didn't have any issues with it on ghwt then I don't see why we shouldn't move to just name 👍